src/blocks/CustomExtendedFeatures/CustomExtendedFeatures.tsx

```diff
@@ -82,9 +82,7 @@ export const CustomExtendedFeatures: React.FC<CustomExtendedFeaturesProps> = ({
                     {button && (
                         <div className={b('button-desktop')}>
                             <Link href={button.href}>
-                                <a>
-                                    <Button size="xl" theme="outlined" text={button.text} />
-                                </a>
+                                <Button size="xl" theme="outlined" text={button.text} />
```

Comment:
Let's correctly use our Button inside next.js Link: https://nextjs.org/docs/pages/api-reference/components/link#if-the-child-is-a-custom-component-that-wraps-an-a-tag

Reply:
done